src/views/search/search.jsx

```diff
@@ -12,6 +12,7 @@ var Input = require('../../components/forms/input.jsx');
 var Button = require('../../components/forms/button.jsx');
 var Tabs = require('../../components/tabs/tabs.jsx');
 var Grid = require('../../components/grid/grid.jsx');
+var Navigation = require('../../redux/navigation.js');
```

Comment:
We should be referring to this in a way that specifies that it's the redux action module, rather than the `Navigation` Component. See how we import [`details`](https://github.com/LLK/scratch-www/blob/develop/src/views/conference/details/details.jsx#L6) on the conference details page for more details on naming.

Additionally, we still need to call the `setSearchTerm` method in the redux module down below once we know what `this.props.searchTerm` actually is.